src/vec_vector.h

```diff
@@ -46,7 +46,7 @@
 namespace quadiron {
 
 template <typename T>
-class Polynomial;
+class Poly;
```

Comment:
Why shorten the name (and making it less explicit)?

`Polynomial` is not that long, especially with autocomplete, and its easier to grep for.

Reply:
this is the current name defined in vec_poly.h

Reply:
removed the comment

Reply:
ho, indeed >_<